src/main/java/com/sngular/kloadgen/property/editor/ValueDeserializerPropertyEditor.java

```diff
@@ -64,7 +64,10 @@ private void fillDeserializer(final JComboBox<String> objectJComboBox) {
             .addUrls(ClasspathHelper.forClass(Deserializer.class))
             .filterInputsBy(new FilterBuilder()
                                 .includePackage("com.sngular.kloadgen.serializer")
-                                .includePackage("io.confluent.kafka.serializers"))
+                                .includePackage("io.confluent.kafka.serializers")
+                                .includePackage("io.apicurio.registry.serde.avro.AvroKafkaDeserializer")
+                                .includePackage("io.apicurio.registry.serde.jsonschema.JsonSchemaKafkaDeserializer")
+                                .includePackage("io.apicurio.registry.serde.protobuf.ProtobufKafkaDeserializer"))
```

Comment:
Is it ok for these literals to be within the code? The serializer also contains the ones corresponding to it

Reply:
Done